src/components/hero/Hero.jsx

```diff
@@ -1,32 +1,71 @@
 import React from 'react'
+import logo from '@assets/logo__cmdf.png'
+import mobile from '@assets/hero__bg_sm.svg'
 import styled from 'styled-components'
-import { SectionContainer } from '@lib/Containers'
 import Button from './Button'
+import background from '@assets/hero__bg.svg'
+import { LAPTOP, fontsize } from '@constants/measurements'
+import { SectionContainerWithBackground as Base } from '@lib/Containers'
 
-const Title = styled.p`
-    text-align: center;
+const SectionContainer = styled(Base)`
+  @import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+
+  font-size: ${() => fontsize(320, 1440, 6, 11.5)};
+  width: 100vw;
+  height: 85.75vw;
+  img {
+    width: 5vw;
+  }
+  .title {
+    font-family: 'Fira Code', monospace;
+  }
+  .date {
+    font-family: 'DM Sans', sans-serif;
+    font-weight: bold;
+  }
+  @media (max-width: ${LAPTOP}) {
+    background-image: url(${mobile});
+    font-size: ${() => fontsize(320, 1440, 6, 34)};
+    height: 250vw;
+    img {
+      width: 16vw;
+    }
+    div {
+      right: 3vw;
+      top: 65vw;
+      width: 40vw;
+    }
+  }
 `
 
 const HeroContainer = styled.div`
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    width: 100vw;
-    height: 100vh;
-    margin: auto;
+  position: relative;
+  right: 4vw;
+  top: 21.5vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 25vw;
+  margin: auto;
+  & > * {
+    margin-bottom: 0.5vw;
+  }
 `
 
 const Hero = ({ hero }) => {
-    const { buttonText, applyActive } = hero
-    return (
-        <SectionContainer>
-            <HeroContainer>
-                <Button enabled={applyActive}>{buttonText}</Button>
-                <p>March 2021</p>
-            </HeroContainer>
-        </SectionContainer>
-    )
+  const { buttonText, applyActive } = hero
+  return (
+    <SectionContainer src={background}>
+      <HeroContainer>
+        <img src={logo} alt="cmd-f logo" />
+        <h1 className="title">Vancouver’s all-female* hackathon</h1>
```

Comment:
I'm not sure about this, do you think we want this to be fetched from firebase? I'm fine either way!

Reply:
Oh I'm not sure, do you think we would need to change it?

Reply:
I would 100% prefer if the texts from line 63 to 65 are pulled from firebase. It would make deprecating the site a lot easier next year.

Reply:
Alright sounds good